mmedit/models/losses/gan_loss.py

```diff
@@ -179,7 +191,8 @@ def compute_zero_padding(kernel_size):
 
         return padding[0], padding[1]
 
-    def get_2d_gaussian_kernel(self, kernel_size, sigma):
+    def get_2d_gaussian_kernel(self, kernel_size: Tuple[int],
+                               sigma: Tuple[int]) -> torch.Tensor:
```

Comment:
`sigma` seems to be a tuple of float, we should change `Tuple[int]` to `Tuple[float, float]` here.